Gemfile

```diff
@@ -10,6 +10,8 @@ end
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.3'
 
+gem 'sprockets', '< 4.0.0'
```

Comment:
Perhaps we should go to `4.0` instead

Reply:
well sure - if you adjust the scss code to mitigate the breaking changes. But for now be aware everyone who sets up a fresh environment using the current master will find this software broken